### PR TITLE
Fix cap calculation

### PR DESCRIPTION
**Pull request by teriaiw**

### CODE REVIEW

**Review by nathanaelseen — APPROVED** [review]
Thanks Teri

File: src/main/java/igrad/model/course/CourseInfo.java [hunk 1]
```diff
@@ -115,11 +115,11 @@ private static int computeCreditsRequired(List<Requirement> requirementList) {
             return Optional.empty();
```
Comment (nathanaelseen) [inline src/main/java/igrad/model/course/CourseInfo.java:115]: good  job Teri, thanks for fixing

Comment (nathanaelseen) [reply src/main/java/igrad/model/course/CourseInfo.java:115]: sorry i had to touch this method abit as well, cos last week i was fixing some cap stuff and some bugs here too, but i'll add @author for you

Comment (teriaiw) [reply src/main/java/igrad/model/course/CourseInfo.java:115]: no probs, thanks!

File: src/main/java/igrad/logic/commands/module/ModuleDoneCommand.java [hunk 2]
```diff
@@ -38,7 +38,7 @@
     public static final String MESSAGE_MODULE_DONE_USAGE = "Parameter(s): MODULE CODE "
         + PREFIX_GRADE + "GRADE\n"
         + "Example: " + MODULE_DONE_COMMAND_WORD + " "
-        + PREFIX_MODULE_CODE + "CS2103T "
```
Comment (nathanaelseen) [inline src/main/java/igrad/logic/commands/module/ModuleDoneCommand.java:41]: thanks Teri, good catch tho

**Pull request merged**